lms/templates/verify_student/test/fake_softwaresecure_response.html

```diff
@@ -56,9 +58,9 @@
 
         $.ajax({
             type: "POST",
-            url: '${results_callback}',
+            url: '${results_callback | n, js_escaped_string}',
             headers: {
-            "Authorization": "${authorization_code}"
+            "Authorization": "${authorization_code | n, js_escaped_string}"
             },
             data: JSON.stringify(data),
```

Comment:
It's the dictionary defined a few lines up. Any parameters passed into the template are being escaped, but there are some hard-coded strings as well (https://github.com/edx/edx-platform/blob/efischer/make_edx_safe_again/lms/templates/verify_student/test/fake_softwaresecure_response.html#L43). Do these need to be escaped as well? They aren't displayed anywhere, and are only used in this test stub.